kolibri/plugins/learn/assets/src/state/actions/main.js

```diff
@@ -177,10 +177,15 @@ function showChannels(store) {
         return;
       }
       const channelRootIds = channels.map(channel => channel.root);
-      ContentNodeResource.getCollection({ ids: channelRootIds }).fetch().then(rootNodes => {
-        const pageState = {
-          rootNodes: _collectionState(rootNodes),
-        };
+      ContentNodeResource.getCollection({ ids: channelRootIds }).fetch().then(channelCollection => {
+        const rootNodes = _collectionState(channelCollection);
+        rootNodes.map(rootNode => {
```

Comment:
if you're just going for the side effect of reassigning `rootNode.thumbnail`, I would use `forEach` instead of `map`.

Reply:
Great point.

Reply:
@jonboiser Updated!